src/lib/containers/download_list_v2/MeetAccessRequirementCard.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useEffect, useState } from 'react'
+import { useErrorHandler } from 'react-error-boundary'
+import { toError } from '../../utils/ErrorUtils'
+import { SynapseSpinner } from '../LoadingScreen'
+import useGetAccessRequirement from '../../utils/hooks/SynapseAPI/useGetAccessRequirement'
+import { AccessRequirement, SelfSignAccessRequirement } from '../../utils/synapseTypes'
+import { Button } from 'react-bootstrap'
+import { Icon } from '../row_renderers/utils'
+import { EASY_DIFFICULTY, MEDIUM_DIFFICULTY, VARIABLE_DIFFICULTY } from '../../utils/SynapseConstants'
+import AccessRequirementList, { SUPPORTED_ACCESS_REQUIREMENTS } from '../access_requirement_list/AccessRequirementList'
+
+export type MeetAccessRequirementCardProps = {
+  accessRequirementId: number
+  count: number
+}
+export const TERMS_OF_USE_TITLE = 'Requires Acceptance of Data-Specific Terms of Use'
+export const SELF_SIGN_TITLE = 'Requires Acceptance of Data-Specific Terms of Use'
+export const ACT_TITLE = 'Requires Approval of Data-Specific Access Requirements'
+export const LOCK_TITLE = 'Access Restricted'
+
+export const MeetAccessRequirementCard:React.FunctionComponent<MeetAccessRequirementCardProps> = (
+  {
+    accessRequirementId,
+    count,
+  }: MeetAccessRequirementCardProps
+) => {
+  const handleError = useErrorHandler()
+  const {
+    data,
+    isFetching,
+    isError,
+    error: newError,
+  } = useGetAccessRequirement(accessRequirementId)
+  const ar:AccessRequirement = data as AccessRequirement
```

Comment:
Could you instead do this?

```suggestion
  const {
    data: ar,
    isFetching,
    isError,
    error: newError,
  } = useGetAccessRequirement(accessRequirementId)
```

I think you lose that it can be potentially undefined if you cast with `as AccessRequirement`. I think you've guarded against it sufficiently with `isError` and `isFetching`, but I'd rather let the TS compiler handle that in general when possible.